src/watchmaker/__init__.py

```diff
@@ -61,14 +79,21 @@ def __init__(
                 level=logging.DEBUG)
             self.logger = logging.getLogger()
             self.logger.info('\n\n\n{0}'.format(datetime.datetime.now()))
-        elif stream:
-            self.logger.error('{0} does not exist'.format(log_path))
+        elif arguments.logger:
+            self.logger.error('{0} does not exist'.format(arguments.log_path))
         else:
             self.logger.warning('Stream logger is not enabled!')
 
         self.logger.info('Parameters:  {0}'.format(self.kwargs))
         self.logger.info('System Type: {0}'.format(self.system))
 
+    def _validate_url(self, url):
+
+        if not validators.url(url):
```

Comment:
Can this just be `return validators.url(url)`? Though the validators docs say this returns an object on failure, wouldn't an object still evaluate to `True`?

https://validators.readthedocs.io/en/latest/#module-validators.url